lib/oelite/oven.py

```diff
@@ -26,7 +26,13 @@
 import time
 
 class OEliteOven:
-    def __init__(self, baker, capacity):
+    def __init__(self, baker, capacity=None):
+        if capacity is None:
+            pmake = baker.config.get("PARALLEL_MAKE")
+            if pmake is None or pmake == "":
+                capacity = 3
+            else:
+                capacity = int(pmake.replace("-j", "")) + 2
```

Comment:
Thinking a bit out of the box. It could be nice if there were a single setting for the amount of parallel "threads" that were allowed/used.

If this parallel thingy were set to 8, that could be

- 8 parallel tasks running do_configure or other sequential stuff
- 1 task running "make -j8"
- 1 taks running "make -j4" and 4 tasks running sequential stuff

and so on.

And this setting should default to something based on the number of available cpu cores.